pkg/backend/display/progress.go

```diff
@@ -1224,7 +1225,7 @@ func (display *ProgressDisplay) getStepInProgressDescription(step engine.StepEve
 	return op.Color() + getDescription() + colors.Reset
 }
 
-func writeString(b *bytes.Buffer, s string) {
+func writeString(b io.StringWriter, s string) {
```

Comment:
Just guessing this is due to the newer compiler helpfully telling us to make this change?

Reply:
I think it’s the linter upgrade possibly combined with the compiler upgrade.